src/app/features/dashboard/dashboard.component.html

```diff
@@ -1,8 +1,54 @@
-<h1>Me, My Shelf & I</h1>
-<h2>Collection</h2>
-<h3>Video Games</h3>
-<div></div>
-<h3>Music</h3>
-<div></div>
-<h3>Movies</h3>
-<div></div>
+<h1>Dashboard</h1>
+<h2>Games</h2>
+<app-loading [height]="120" *ngIf="loading; else gamesCollection" />
+<ng-template #gamesCollection>
+  <div
+    *ngIf="collection.games.length > 0; else noGames"
+    class="media-selection">
+    <ng-template ngFor let-game [ngForOf]="collection.games">
+      <app-game-card [data]="game.general" [uid]="uid" />
+    </ng-template>
+  </div>
+  <ng-template #noGames>
+    <div class="no-content-disclaimer">
+      <span>You haven't added any games to your collection.</span>
+      <a routerLink="/games">Browse for games here!</a>
+    </div>
+  </ng-template>
+</ng-template>
+
+<h2>Movies</h2>
+<app-loading [height]="120" *ngIf="loading; else moviesCollection" />
+<ng-template #moviesCollection>
```

Comment:
ah I see what you're doing with the loading spinner.
Before I saw it and the different heights and assumed it's a loding spinner per page but it's per component so it's good.

A good spot to refactor after the PR though 👍